asserts/extkeypairmgr_test.go

```diff
@@ -0,0 +1,267 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2021 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package asserts_test
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+
+	. "gopkg.in/check.v1"
+
+	"github.com/snapcore/snapd/asserts"
+	"github.com/snapcore/snapd/asserts/assertstest"
+	"github.com/snapcore/snapd/testutil"
+)
+
+type extKeypairMgrSuite struct {
+	pgm *testutil.MockCmd
+
+	defaultPub *rsa.PublicKey
+	modelsPub  *rsa.PublicKey
+}
+
+var _ = Suite(&extKeypairMgrSuite{})
+
+func (s *extKeypairMgrSuite) SetUpSuite(c *C) {
+	tmpdir := c.MkDir()
+	k1, err := rsa.GenerateKey(rand.Reader, 4096)
```

Comment:
Out of curiosity, why big keys like this here in the testing code? I guess it does not really matter much (it's just tests), was just wondering as e.g. in httputil we only use 768

Reply:
because our signing code checks for this size and fails otherwise, what we could do it's use a non-random generator I suppose